### PR TITLE
adding support for debian 6

### PR DESCRIPTION
**Pull request by flosell**
This adds support for debian by adding the ubuntu ppa as an apt-source. tested with debian 6.0.6. I didn't do regression-testing on ubuntu though

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -8,15 +8,30 @@
...
   case $::operatingsystem {
-    debian, ubuntu: {
+    debian: {
       include apt
+      
+      apt::source { 'webupd8team': 
+	location          => "http://ppa.launchpad.net/webupd8team/java/ubuntu",
+        release           => "precise",
+        repos             => "main",
+        #required_packages => "debian-keyring debian-archive-keyring",
```
Comment (smerchek) [inline manifests/init.pp]: Since this is commented out, is it ever needed? If not, let's just delete it.

**Comment by smerchek** [comment]
Thanks for this pull. I'm downloading a debian vagrant box so that I can verify. Other than a couple things, this looks good though. Thanks for the pull!

**Comment by smerchek** [comment]
It worked in my vagrant environment! I'm ready to merge this once the commented line is explained.

File: manifests/init.pp (hunk 1 above)
Comment (flosell) [inline manifests/init.pp]: Oh, yeah, that line should not be needed, just forgot to remove it.

**Pull request merged**